javascript/test/__snapshots__/collection.test.ts.snap

```diff
@@ -0,0 +1,159 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`collection can generate collection declaration code 1`] = `
+"function anonymous(
+) {
+return function(fury, external) {
+  const br = fury.binaryReader;
+  const bw = fury.binaryWriter;
+  const cr = fury.classResolver;
+  const rr = fury.referenceResolver;
+  
+  const readInner = (fromRef) => {
+    
+    const flags_2 = br.uint8();
+    br.skip(2);
+    const len_1 = br.varUInt32();
+    const result_0 = new Array(len_1);
+    
+    if (fromRef) {
+      rr.reference(result_0)
+    }
+    
+    if (flags_2 & 1) {
```

Comment:
@chaokunyang  This snapshot is generated by a typed collection declared with `Type.array(Type.string())`, which is equivalent to `Array<String>` in Java.